geo/src/algorithm/map_coords.rs

```diff
@@ -745,7 +726,10 @@ pub(crate) mod deprecated {
     use super::*;
 
     /// Map a fallible function over all the coordinates in a geometry, returning a Result
-    #[deprecated(since = "0.21.0", note = "use `MapCoords::try_map_coords` instead")]
+    #[deprecated(
+        since = "0.21.0",
+        note = "use `MapCoords::try_map_coords` which takes a `Coordinate` instead of an (x,y) tuple"
+    )]
     pub trait TryMapCoords<T, NT, E> {
```

Comment:
Note, for the deprecated methods, I left the signatures as `(T, T)` tuples, figuring they'll go away soon and this way we can better have the compiler tell the users what to do.